modelcard.md

```diff
@@ -18,20 +18,26 @@ metrics:
 model-index:
 - name: {model_id}
   results:
-  - task: 
-      type: {task_type}  # Required. Example: automatic-speech-recognition
-      name: {task_name}  # Optional. Example: Speech Recognition
+  - task:
+      type: {task_type}             # Required. Example: automatic-speech-recognition
+      name: {task_name}             # Optional. Example: Speech Recognition
     dataset:
-      type: {dataset_type}  # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
-      name: {dataset_name}  # Required. Example: Common Voice zh-CN
-      args: {arg_0}         # Optional. Example: zh-CN
+      type: {dataset_type}          # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
+      name: {dataset_name}          # Required. Example: Common Voice (French)
+      config: {dataset_config}      # Optional. Example: fr
+      split: {dataset_split}        # Optional. Example: test
```

Comment:
Interesting idea :) 

Are you aware of any examples where one trains a model on a dataset config, but evaluates on a different one? For the splits, I guess the simplest change would be to have something like:

```
  splits:
    train_split: train
    eval_split: test
```

This would then align with the `train-eval-index` metadata that we're planning to add to dataset cards in https://github.com/huggingface/autonlp-backend/issues/414

Reply:
One example that comes to mind is LibriSpeech that has a `clean` config for training/finetuning, and `clean`+`other` configs (with `clean-dev`/`clean-test`/`other-dev`/`other-test` splits) for evaluation, but it's particularly messy in itself IMO.
So the addition of training splits should be enough I think!

Reply:
`model-index` is for eval results, are we sure we want to also add training info in there? might be a bit confusing

The alternative would be to come up with a different training-specific ontology (even though they might look very similar)

Reply:
> `model-index` is for eval results, are we sure we want to also add training info in there? might be a bit confusing

I agree, and think it makes most sense to just report the split that the model was evaluated on.